### DISCUSSION TITLE
Blender BIM installation

### DISCUSSION THREAD

**Discussion by guieliote** [IfcOpenShell/IfcOpenShell#3972] [General]
Hi everyone, 
I am new to Blender/BlenderBIM, so this might be a very trivial question. 
I was trying to install the latest add-on "BlenderBim", and when I finished the installation, instead of seeing "Import-Export: BlenderBim", like it's been shown here, I saw "System: BlenderBim". I also can't see the information about the IFC in the scene panel...
Does anyone here know what's causing that?

**Comment by theoryshaw** [comment]
Hi @guieliote 
see: https://community.osarch.org/discussion/1733/installing-blenderbim-problem

If a relatively newbie, would suggest asking questions on the https://community.osarch.org/ 

Cheers.

**Reply by guieliote** [reply]
Thank you so much for your response!
